libraries/AP_CustomRotations/AP_CustomRotations_params.cpp

```diff
@@ -0,0 +1,32 @@
+#include "AP_CustomRotations.h"
+#include <AP_Vehicle/AP_Vehicle_Type.h>
+
+#if !APM_BUILD_TYPE(APM_BUILD_AP_Periph)
```

Comment:
Would be nice to have an `AP_CUSTOMROTATIONS_ENABLED` so that if a Periph really did want to allow custom rotations they could.